client/__tests__/QuestionsAndAnswers.test.js

```diff
@@ -7,14 +7,18 @@ import QACardAnswers from '../src/components/QA/QACardAnswers.jsx';
 import QAAddAnswerModal from '../src/components/QA/QAAddAnswerModal.jsx';
 import QAAddQuestionModal from '../src/components/QA/QAAddQuestionModal.jsx';
 import { productInfoMock } from '../src/mockData/productInfoMock.js';
+import { questionsMock, answersMock } from '../src/mockData/questionsMock.js';
 import axios from 'axios';
 
+
 beforeEach(() => {
-  render(<QuestionsAndAnswers productInfoMock={productInfoMock}/>);
-  render(<QACardAnswers />);
-  axios.get('/qa/questions/')
-    .then(response => response)
-    .catch(err => err);
+  render(<QuestionsAndAnswers productInfo={productInfoMock} questionsInfo={questionsMock}/>,
```

Comment:
* Something about the change in tests is leading to a new error during the onComponentRender process. This error is probably happening because there is not environment key in the CI for github so we don't notice this error running tests locally. 
* Something about the way the error is being handled in the console.error() catch statement is happening after the test should be done so perhaps it has something to do with rendering multiple components beforeEach() test. 
* I wonder if you can accomplish what you need by rendering the topmost component since it will pass down what the subordinate ones need. The other option would be trying to only render and clear each component within one test and see if that changes the behavior. Unfortunately you may need to push the change to github to test. 
* You might be able to recreate the same issue if you break your local env. variable connection to your API keys.

Reply:
@NylonEric Changing the error catch block allowed the tests to pass now. These specific lines were updated to allow me to use mock data instead of the server data as before.